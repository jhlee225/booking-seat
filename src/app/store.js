import { configureStore } from "@reduxjs/toolkit";
import loginReducer from "../features/login/loginSlice";
import seatReducer from "../features/seat/seatSlice";
import menuReducer from "../features/menu/menuSlice";
import bookReducer from "../features/booking/BookSlice";
import timeReducer from "../features/time/timeSlice";
import signUpReducer from "../features/signup/signupSlice";

export default configureStore({
  reducer: {
    login: loginReducer,
    seat: seatReducer,
    menu: menuReducer,
    time: timeReducer,
    book: bookReducer,
    signUp: signUpReducer,
  },
});
