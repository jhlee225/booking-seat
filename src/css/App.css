@import "reset.css";
@import "header.css";
@import "menu.css";
@import "login.css";
@import "signUp.css";
@import "seats.css";
@import "book.css";
@import "time.css";
@import "footer.css";

@media screen and (orientation: portrait) {
  body {
    font-family: sans-serif;
    width: 100vmin;
    height: 100vmax;
  }
}

@media screen and (orientation: landscape) {
  body {
    font-family: sans-serif;
    width: 100vmax;
    height: 100vmin;
  }
}

a {
  text-decoration: none;
  color: white;
}

@media screen and (orientation: portrait) {
  .App {
    height: 87vmax;
  }
}

@media screen and (orientation: landscape) {
  .App {
    height: 87vmin;
  }
}
.App {
  background: linear-gradient(
      to top,
      slategrey,
      rgb(115, 135, 155, 0.5) 1%,
      rgb(115, 135, 155, 0.5) 99%,
      slategrey
    ),
    url("./images/book.jpg");
  background-position: center;
  background-size: 100% 100%;
  display: flex;
  justify-content: center;
  align-items: center;
}

.bookingWrap,
.SignUpWrap,
.loginWrap {
  margin: 10px;
  padding: 20px 5px;
  display: flex;
  flex-direction: column;
  border-radius: 10px;
  background-color: rgb(245, 245, 245, 0.9);
}
