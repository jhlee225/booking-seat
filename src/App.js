import React from "react";
import "./css/App.css";
import { HashRouter as Router, Route } from "react-router-dom";
import { Header } from "./common/Header";
import { Menu } from "./features/menu/menu";
import { Login } from "./features/login/login";
import { Seats } from "./features/seat/seat";
import { Footer } from "./common/Footer";

function App() {
  return (
    <>
      <Header />
      <div className="App">
        <Router>
          <Route exact path="/" component={Menu}></Route>
          <Route exact path="/Login" component={Login}></Route>
          <Route exact path="/Seat" component={Seats}></Route>
        </Router>
      </div>
      <Footer />
    </>
  );
}

export default App;
