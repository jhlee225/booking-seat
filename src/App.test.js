import React from "react";
import { render } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "./app/store";
import App from "./App";

it("render App", () => {
  const appRendered = render(
    <Provider store={store}>
      <App />
    </Provider>
  );

  expect(appRendered.container).toMatchSnapshot();
});
