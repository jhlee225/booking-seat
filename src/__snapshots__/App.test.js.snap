// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App 1`] = `
<div>
  <header
    class="loginHeader"
  >
    <a
      href="#/"
    >
      <span>
        좌석 예약 시스템
      </span>
    </a>
  </header>
  <div
    class="App"
  >
    <div
      class="menubtns"
    >
      <a
        href="#/Seat"
      >
        <div
          class="menuBtn"
        >
          좌석보기
        </div>
      </a>
      <a
        href="#/Login"
      >
        <div
          class="menuBtn"
        >
          로그인
        </div>
      </a>
      <a
        href="#/SignUp"
      >
        <div
          class="menuBtn"
        >
          회원가입
        </div>
      </a>
    </div>
  </div>
  <footer
    class="loginFooter"
  >
    <span>
      FrontEnd 이재혁
    </span>
    <span>
      주태종 BackEnd
    </span>
  </footer>
</div>
`;
